README.md

```diff
@@ -24,6 +24,36 @@ It generates the custom chaos experiments with some default Pre & Post Chaos Che
 
 Refer [Litmus-SDK](https://github.com/litmuschaos/litmus-python/blob/master/contribute/developer-guide/README.md) for more details
 
+### Overview
+
+Litmus-Python chaos experiments are fundamental units within the LitmusChaos architecture. Users can choose between readily available chaos experiments or create new ones to construct a required Chaos Workflow.
+
+For more about working of LitmusChaos [refer](https://litmuschaos.github.io/litmus/).
```

Comment:
To know more about LitmusChaos experiments [refer to this](https://litmuschaos.github.io/litmus/).